xtask/src/ast_src.rs

```diff
@@ -402,107 +679,795 @@ pub(crate) const AST_SRC: AstSrc = AstSrc {
             T![;]
         }
 
+        /// Inherent and trait impl definition.
+        /// Includes all of its inner and outer attributes.
+        ///
+        /// ```
+        /// ❰
+        ///     #[attr]
+        ///     unsafe impl<T> !Foo for Bar where T: Debug {
+        ///         #![inner_attr]
+        ///         // ...
+        ///     }
+        /// ❱
+        /// ```
+        ///
+        /// [Reference](https://doc.rust-lang.org/reference/items/implementations.html)
         struct ImplDef: TypeParamsOwner, AttrsOwner {
             T![default],
-            T![const],
+            T![const], // TODO: wat?
```

Comment:
https://github.com/rust-lang/rust/blob/6af388b25050bca26710be7e4030e17bf6d8d2f7/src/test/ui/rfc-2632-const-trait-impl/const-and-non-const-impl.rs

Reply:
Thanks, added const to the example in 09c438b47eb035e84bc013a3e78fae9f938e9b85